package.json

```
@@ -38,7 +38,7 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
-    "@dashevo/dapi-client": "^0.9.0-dev.6",
+    "@dashevo/dapi-client": "^0.12.0",
```

Should we change to this since dapi-client "minor" changes can be still be "major" (like we have done in DashJS and some other repos, I think)?
```suggestion
    "@dashevo/dapi-client": "~0.12.0",
```

---

Thanks ! :)